### PR TITLE
Popin ad extension document updated.

### PR DESCRIPTION
**Pull request by popinuser**
# Popin ad extenstion document updated.

Fixed because document tags were not closed.

### CODE REVIEW

**Comment by popinuser** [comment]
It is a small fix, please, please review.

File: ads/popin.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ limitations under the License.
     height=568
     layout=responsive
     heights="(min-width:1907px) 39%, (min-width:1200px) 46%, (min-width:780px) 64%, (min-width:480px) 98%, (min-width:460px) 167%, 196%"
-    data-mediaid="popin_amp"
+    data-mediaid="popin_amp">
```
Comment (zhouyx) [inline ads/popin.md]: I see you change the example in `examples/ads.amp.html`. let's also make the change here, so they look the same.

Comment (jridgewell) [inline ads/popin.md]: Afraid I don't follow. Can you recheck?

File: examples/ads.amp.html [hunk 2]
```diff
@@ -1046,7 +1046,7 @@ <h2 id="plista">Plista responsive widget</h2>
   </amp-embed>
 
   <h2 id="popin">popIn native ad</h2>
-  <amp-ad width="300" height="568"
+  <amp-ad width=300 height=568
```
Comment (zhouyx) [inline examples/ads.amp.html]: no need to remove ""

File: ads/popin.md (hunk 1 above)
Comment (zhouyx) [inline ads/popin.md]: Oh it's just styling. I mean
```
 <amp-ad width="300" height="568"
      type="popin"
      layout=responsive
      heights="(min-width:1907px) 39%, (min-width:1200px) 46%, (min-width:780px) 64%, (min-width:480px) 98%, (min-width:460px) 167%, 196%"
      data-mediaid="popin_amp">
  </amp-ad>
```
(copied from the example provided in `ads.amp.html`)

**Comment by popinuser** [comment]
@zhouyx Thank you review.	I replaced the removed double quotes.

**Comment by zhouyx** [comment]
LGTM. @erwinmombay would you please force merge the PR?

**Pull request merged**